src/main/kotlin/lotto/domain/LottoTickets.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+data class LottoTickets(
+    private val lottoTickets: List<LottoTicket>
+) {
+    fun get() = lottoTickets.toList()
+
+    operator fun plus(value: LottoTickets): LottoTickets =
+        LottoTickets(lottoTickets + value.lottoTickets)
+
+    fun compareNumber(winningLotto: WinningLotto): LottoResults =
+        lottoTickets.asSequence()
+            .mapNotNull { lottoTicket ->
+                lottoTicket.toLottoPrize(winningLotto)
+            }.groupingBy { it }.eachCount()
+            .map { (LottoPrize, count) -> LottoResult(LottoPrize, PositiveNumber(count)) }
```

Comment:
```suggestion
            .map(::LottoResult)
```
`LottoResult` 에 생성자를 하나 더 넣어주면 사용하는쪽에서는 간결하게 표현할 수 있을것 같아요.
부 생성자를 적절하게 사용하면 이처럼 객체를 사용하는쪽에 가독성을 높일 수 있습니다. 😀

Reply:
부생성자를 이용하는걸 자꾸 놓치네요 ㅎㅎ 노력해보겠습니다!